folio-spring-i18n/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.folio</groupId>
+    <artifactId>folio-spring-support</artifactId>
+    <version>7.3.0-SNAPSHOT</version>
+  </parent>
+
+  <name>folio-spring-i18n</name>
+  <artifactId>folio-spring-i18n</artifactId>
+  <description />
+
+  <properties>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
```

Comment:
You can probably replace these properties with `<maven.compiler.release>17</maven.compiler.release>` - release mostly took the place of source/target a while back